src/common/tokenHelpers.ts

```diff
@@ -91,3 +93,7 @@ export const createTokenIdFromContractAddress = (token: EdgeToken): string => {
   }
   return cleanLocation.contractAddress
 }
+
+export const createEvmTokenId = (token: EdgeToken): string => {
+  return normalizeAddress(createTokenIdFromContractAddress(token))
```

Comment:
This function is named normalizeAddress yet we're using it to get a "token ID". Weird. Oh well.